apps/src/lab2/levelEditors/aichatSettings/ModelCardFields.tsx

```diff
@@ -1,16 +1,17 @@
 import React, {useContext} from 'react';
 import MultiItemInput from './MultiItemInput';
-import moduleStyles from './edit-ai-customizations.module.scss';
+import moduleStyles from './edit-aichat-settings.module.scss';
 import {
   MAX_ASK_ABOUT_TOPICS,
   MODEL_CARD_FIELDS_AND_LABELS,
 } from '@cdo/apps/aichat/views/modelCustomization/constants';
 import {UpdateContext} from './UpdateContext';
 
 const ModelCardFields: React.FunctionComponent = () => {
-  const {setModelCardPropertyValue, aiCustomizations} =
+  const {setModelCardPropertyValue, aichatSettings: aiCustomizations} =
```

Comment:
Does this create an alias for `aichatSettings`? If so, could you explain why you did this instead of just using `aichatSettings`? If not, could you explain what this does?

Reply:
Oh good catch, this is just something I forgot to clean up after renaming variables. Will fix!